mods/cnc/weapons/smallcaliber.yaml

```diff
@@ -185,7 +185,7 @@ APCGun.AA:
 	Report: gun20.aud
 	ValidTargets: Air
 	Projectile: Bullet
-		Speed: 1c682
+		Speed: 2
```

Comment:
What?!

Unless you actually aimed this to be evaded literally by anything moving faster than a snail, this is wrong.